contracts/samples/bls/BLSSignatureAggregator.sol

```diff
@@ -104,14 +111,13 @@ contract BLSSignatureAggregator is IAggregator {
         return BLSOpen.hashToPoint(BLS_DOMAIN, abi.encodePacked(userOpHash));
     }
 
-    // helper for test
     function getUserOpHash(UserOperation memory userOp) public view returns (bytes32) {
         bytes32 publicKeyHash = _getPublicKeyHash(getUserOpPublicKey(userOp));
         return _getUserOpHash(userOp, publicKeyHash);
     }
 
     function _getUserOpHash(UserOperation memory userOp, bytes32 publicKeyHash) internal view returns (bytes32) {
-        return keccak256(abi.encode(internalUserOpHash(userOp), publicKeyHash, address(this), block.chainid));
+        return keccak256(abi.encode(internalUserOpHash(userOp), publicKeyHash, address(this), block.chainid, entryPoint));
```

Comment:
why hash aggregator at all? doesn't EP replaces it?

Reply:
Is there a reason not to hash it? I'm usually inclined to prevent malleability even if we can't think of a way to attack it right now.